### PR TITLE
figure autoscale

### PR DESCRIPTION
**Pull request by aloctavodia**
yet another try

### CODE REVIEW

File: arviz/plots/plot_utils.py [hunk 1]
```diff
@@ -42,31 +42,42 @@ def _scale_fig_size(figsize, textsize, rows=1, cols=1):
...
+    if isinstance(rc_titlesize, str):
+        rc_titlesize = 16
+    if isinstance(rc_xt_labelsize, str):
+        rc_xt_labelsize = 14
+
     if figsize is None:
         width, height = rc_width, rc_height
-        width = width * cols
-        height = height * rows
+        sff = 1 if (rows == cols == 1) else 1.15
```
Comment (canyon289) [inline arviz/plots/plot_utils.py]: How are these magic numbers decided?

Comment (aloctavodia) [reply arviz/plots/plot_utils.py]: I was trying to make that for a figure with multiple subplots each subplot has the same size as a single figure.

Comment (canyon289) [reply arviz/plots/plot_utils.py]: Got it!

**Comment by ColCarroll** [comment]
Merging! It will be cool when #330 lands, and we can compare a bunch of plots for a change like this.

**Pull request merged**